Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,53 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
+    {
+        readonly ICollection<string> _repositories = new List<string>();
+
+        public IEnumerator<string> GetEnumerator()
+        {
+            return _repositories.GetEnumerator();
+        }
+
+        IEnumerator IEnumerable.GetEnumerator()
+        {
+            return GetEnumerator();
+        }
+
+        public void Add(string owner, string name)
```

Comment:
I actually had both in there, but decided to delete the overload. I guess it doesn't hurt to have it in, since we still do the same, full, validation :smile: